packages/editor/src/components/rich-text/editable.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * External dependencies
+ */
+import { isEqual } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Component, createElement } from '@wordpress/element';
+import { BACKSPACE, DELETE } from '@wordpress/keycodes';
+
+/**
+ * Internal dependencies
+ */
+import { diffAriaProps, pickAriaProps } from './aria';
+
+/**
+ * Browser dependencies
+ */
+
+const { userAgent } = window.navigator;
+
+/**
+ * Applies a fix that provides `input` events for contenteditable in Internet Explorer.
+ *
+ * @param {Element} editorNode The root editor node.
+ *
+ * @return {Function} A function to remove the fix (for cleanup).
+ */
+function applyInternetExplorerInputFix( editorNode ) {
```

Comment:
Is this something that was done by TinyMCE, is this the only browser quirk done by TinyMCE?

Reply:
No this isn't something that was done by TinyMCE. This code is currently in master too. It's to fix the `input` event for IE11. Original code was added by @brandonpayton I believe.